PowerEditor/src/Parameters.cpp

```diff
@@ -72,7 +72,7 @@ struct ScintillaKeyDefinition
 */
 static const WinMenuKeyDefinition winKeyDefs[] =
 {
-	// V_KEY,    COMMAND_ID,                                    Ctrl,  Alt,   Shift, cmdName
+	// V_KEY,    COMMAND_ID,                                   Ctrl,  Alt,   Shift, cmdName

```

Comment:
PR should not contains reformatting, ie. remove/add any space and TAB :
https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/CONTRIBUTING.md#guidelines-for-pull-requests